aws/arctic-wolf/arctic_wolf.tf

```diff
@@ -0,0 +1,212 @@
+  restrict_public_buckets = true
+}
+
+resource "aws_cloudtrail" "default" {
+  count                 = var.create && var.cloudtrail == null ? 1 : 0
+  name                  = join("-", [local.module_prefix])
+  s3_bucket_name        = aws_s3_bucket.default[0].id
+  is_multi_region_trail = true
+}
+
+resource "aws_cloudformation_stack" "cloudtrail" {
+  count        = var.create ? 1 : 0
+  name         = join("-", [local.module_prefix])
+  capabilities = ["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM"]
+
+  parameters = {
+    cloudtrailTrail  = var.cloudtrail == null ? aws_cloudtrail.default[0].name : var.cloudtrail
+    logRetentionDays = 30
+  }
+
+
+  template_url = "https://s3.amazonaws.com/arcticwolf-public/install/aws-templates/latest/primary_region_template.json"
+}
+
+data "aws_guardduty_detector" "default" {
+  count = var.create && var.enable_guardduty_logging && var.guardduty_detector_id == null ? 1 : 0
+}
+
+resource "aws_cloudformation_stack" "guardduty" {
+  count        = var.create && var.enable_guardduty_logging ? 1 : 0
+  name         = join("-", [local.module_prefix, "guardduty"])
+  capabilities = ["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM"]
+
+  parameters = {
+    GuardDutyDetectorID = coalesce(var.guardduty_detector_id, data.aws_guardduty_detector.default[0].id)
+  }
+
+
+  template_url = "https://s3.amazonaws.com/arcticwolf-public/install/aws-templates/latest/awn_guardduty_template.json"
+
+  depends_on = [
+    aws_cloudformation_stack.cloudtrail[0],
+  ]
+}
+
+resource "aws_cloudformation_stack" "vpc_flow_log" {
+  count        = var.create && var.vpc_id != null ? 1 : 0
+  name         = join("-", [local.module_prefix, "vpc-flow-log"])
+  capabilities = ["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM"]
+
+  parameters = {
+    vpcId = var.vpc_id
+  }
+
+
+  template_url = "https://s3.amazonaws.com/arcticwolf-public/install/aws-templates/latest/vpcflowlogs_template.json"
+
+  depends_on = [
+    aws_cloudformation_stack.cloudtrail[0],
+  ]
+  }
+
+resource "aws_cloudformation_stack" "vpc_flow_log_group" {
+  count        = var.create && var.vpc_flow_log_group_name != null ? 1 : 0
+  name         = join("-", [local.module_prefix, "vpc-flow-log-group"])
+  capabilities = ["CAPABILITY_IAM", "CAPABILITY_NAMED_IAM"]
+
+  parameters = {
+    filterPattern = var.flow_log_filter_pattern
+    logGroupName  = var.vpc_flow_log_group_name
+  }
+
+
+  template_url = "https://s3.amazonaws.com/arcticwolf-public/install/aws-templates/latest/cloudwatch_logs_template.json"
+
+  depends_on = [
+    aws_cloudformation_stack.cloudtrail[0],
+  ]
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Outputs
+# ----------------------------------------------------------------------------------------------------------------------
+
+output "cloudtrail" {
+  value       = var.cloudtrail == null ? aws_cloudtrail.default[0].name : null
+  description = ""
+}
+
+output "topicArn" {
```

Comment:
Should this be snake case?

Reply:
Modified name to correctly identify output value